docs/sources/release-notes/v2.1.md

```diff
@@ -0,0 +1,39 @@
+
+- **`Remote` ruler mode for improved rule evaluation performance**: We've added a `remote` mode for the Grafana Mimir ruler, in which the ruler delegates rule evaluation to the [query-frontend]({{< relref "../operators-guide/architecture/components/query-frontend/index.md" >}}) rather than evaluating rules directly within the ruler process itself. This allows recording and alerting rules to benefit from the query parallelization techniques implemented in the query-frontend (like query sharding). `Remote` mode is considered experimental and is off by default. To enable, see [remote ruler]({{< relref "../operators-guide/architecture/components/ruler/#remote" >}}).
+
+- **Per-tenant custom trackers for monitoring cardinality**: In Grafana Mimir 2.0, we introduced a [custom tracker feature]({{< relref "../operators-guide/configuring/configuring-custom-trackers.md" >}}) that allows you to track the count of active series over time that match a specific label matcher. In Grafana Mimir 2.1, we've made it possible to configure custom trackers via the [runtime configuration file]({{< relref "../operators-guide/configuring/about-runtime-configuration.md" >}}). This means you can now define different trackers for each tenant in your cluster and modify those trackers without an ingester restart.
+
+- **Reduce cardinality of Grafana Mimir's `/metrics` endpoint**: While Grafana Mimir does a good job of exposing a relatively small number of series about its own state, this number can tick up when running Grafana Mimir clusters with high tenant counts or high active series counts. To reduce this number (and the accompanying cost of scraping and storing these time series), we've made [several optimizations](https://github.com/grafana/mimir/issues/1750). Our improvements have already reduced series count on the `/metrics` endpoint of one of our internal clusters by 11%, and we expect an even greater reduction on our bigger clusters once this rolls out there.
+
+## Upgrade considerations
+
+We've updated the default values for 2 parameters in Grafana Mimir to give users better out-of-the-box performance:
+
+- We've changed the default for `-blocks-storage.tsdb.isolation-enabled` from `true` to `false`. We've marked this flag as deprecated and will remove it, setting the value permanently to `false`, in 2 releases. Our decision to do this came from our experience running our [1 billion series load test](https://grafana.com/blog/2022/04/08/how-we-scaled-our-new-prometheus-tsdb-grafana-mimir-to-1-billion-active-series/#prometheus-tsdb-enhancements), where we saw that disabling this setting reduced ingester 99th percentile latency by 90%.
```

Comment:
Perhaps we should point out that TSDB isolation feature (within single ingester) doesn't bring any benefit in our architecture, where single push request is distributed to many ingesters. Mimir didn't provide isolation guarantees even with this option enabled.

Reply:
Oh, I wouldn't add anything to the blog post, just to release notes. But if you don't think it's necessary, that's fine. You're correct that it's explained in the blog post already.

Reply:
I tried again! lemme know what you think.